### PR TITLE
Feat : Added Card Component

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Card
Added card Component.

### CODE REVIEW

File: assets/Css/ComponentsStyles.css [hunk 1]
```diff
@@ -581,3 +581,121 @@ cursor: pointer;
...
+}
+.card__button--primary{
+  margin-right: 0.2rem;
+  background-color: blueviolet;
+  color: white;
+}
+.card__button--secondary{
+margin-left: 0.2rem;
+background-color: transparent;
+color: blueviolet;
```
Comment (mariyasada) [inline assets/Css/ComponentsStyles.css:625]: use color variables

**Review by mariyasada — APPROVED** [review]
use color variables

**Review by ShreyashMahajan — APPROVED** [review]
approved

File: assets/Css/ComponentsStyles.css (hunk 1 above)
Comment (SaurabhChawda) [reply assets/Css/ComponentsStyles.css:625]: Okay

**Pull request merged**